pydanticrud/backends/dynamodb.py

```diff
@@ -112,7 +115,7 @@ def index_definition(index_name, keys, gsi=False):
 
 class DynamoSerializer:
     def __init__(self, schema):
-        self.properties = schema["properties"]
+        self.properties = schema.get("properties")
```

Comment:
Are we concerned at all about the None value possibility here?

Reply:
This is a simple change in future if we add DynamotypeSerializer as separate serializer to pydanticrud this will be helpful as it won't expect the properties from that.